index.bs

```diff
@@ -1783,8 +1783,8 @@ Charter Refinement</h3>
 	with the goal of achieving [=consensus=] on the proposal.
 	The <dfn>Chartering Facilitator</dfn>--
 	who is chosen (and <em class=rfc2119>may</em> be replaced) by the [=Team=]--
-	acts as [=Chair=] for the [=charter refinement=] process,
-	and is responsible for seeking community consensus among those participating in the refinement process.
+	is responsible for seeking community [=consensus=] among those participating in the refinement process
+	and making decisions reflecting that [=consensus=].
 
```

Comment:
If this pull request is accepted, it would remove the text that PR #997 was proposing to amend. Here's a possible amendment to this PR that introduces an analogous change, which clarifies that decisions in the absence of consensus can be made, and how they are made.

```suggestion
	In cases where [=consensus=] cannot be found,
	the [=Chartering Facilitator=] <em class=rfc2119>may</em> ask the [=Team=] to make a [=Team Decision=],
	and <em class=rfc2119>must</em> document the rationale for the decision.

```